src/System.Xml.XmlSerializer/tests/XmlSerializerTests.cs

```diff
@@ -396,7 +396,7 @@ private static void VerifyXElementObject(XElement x1, XElement x2, bool checkFir
     public static void Xml_WithXElementWithNestedXElement()
     {
         var original = new WithXElementWithNestedXElement(true);
-        var actual = SerializeAndDeserialize<WithXElementWithNestedXElement>(original, "<?xml version=\"1.0\"?>\r\n<WithXElementWithNestedXElement xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">\r\n  <e1>\r\n    <ElementName1 Attribute1=\"AttributeValue1\">\r\n      <ElementName2 Attribute2=\"AttributeValue2\">Value2</ElementName2>\r\n    </ElementName1>\r\n  </e1>\r\n</WithXElementWithNestedXElement>");
+        var actual = SerializeAndDeserialize<WithXElementWithNestedXElement>(original, "<?xml version=\"1.0\"?>" + Environment.NewLine + "<WithXElementWithNestedXElement xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">\r\n  <e1>\r\n    <ElementName1 Attribute1=\"AttributeValue1\">\r\n      <ElementName2 Attribute2=\"AttributeValue2\">Value2</ElementName2>\r\n    </ElementName1>\r\n  </e1>" + Environment.NewLine + "</WithXElementWithNestedXElement>");
```

Comment:
That would still be an issue; the newlines need to change based on where the binary runs, and in this case there's just one binary used everywhere. I can certainly split these up across multiple lines, just not inside verbatim strings.